### DISCUSSION TITLE
Update link in OC versions <v2.5 is unreadable and unusable

### DISCUSSION THREAD

**Discussion by RachelKarchin** [KarchinLab/open-cravat#261] [Usability & Feature Requests]
![image](https://github.com/KarchinLab/open-cravat/assets/12953556/f345e7c0-f3bb-4412-b761-48b805898ec4)

**Comment by skanderson** [comment]
The update notification was cleaned up in [release 2.5.0](https://github.com/KarchinLab/open-cravat/releases/tag/2.5.0). It will now look something like this:
![image](https://github.com/KarchinLab/open-cravat/assets/124068392/bf0223c0-f731-4829-ae61-212f3dbfbd9e)

**For Users on Versions Prior to 2.5.0**
If you are affected by this bug and need to update the system, the most common way to update is to use pip with the command:
```bash
pip install open-cravat --upgrade --no-cache-dir
```
If you used an OpenCravat installer, or would like more information, see our [System Update Documentation](https://open-cravat.readthedocs.io/en/latest/Update-Instructions.html)
